README.md

```diff
@@ -707,6 +708,26 @@ type Name = $NonMaybeType<MaybeName>;
 
 [⇧ back to top](#flows-utility-types)
 
+### `Class<T>`
+
+Given a type T representing instances of a class C, the type Class<T> is the type of the class C  
+https://flow.org/en/docs/types/utilities/#toc-class  
+\* Differs from original Flow's util - implements only constructor part and won't include any static members. Additionally classes in Typescript are not treated as nominal 
+
+**Usage:**
+
+```ts
+import { Class } from 'utility-types';
+
+class Store {}
+
+function makeStore(storeClass: Class<Store>) {
```

Comment:
Please also include return type `Store` here, so it is more obvious for the user what it should be

Reply:
After that small change I'll merge